pkg/cmd/tests/tests_run.go

```diff
@@ -412,16 +413,25 @@ func (o *RunOptions) run(ctx context.Context, streams genericclioptions.IOStream
 	commonArgs = append(commonArgs, fmt.Sprintf("--%s=%v", parallelServerAddressFlagKey, server.Address()))
 	commonArgs = append(commonArgs, fmt.Sprintf("--%s=%v", cmdutil.FlagLogLevelKey, o.ParallelLogLevel))
 
+	foundRandomSeedArg := false
+	for i := range commonArgs {
+		if strings.HasPrefix(commonArgs[i], fmt.Sprintf("--%s", randomSeedFlagKey)) {
+			foundRandomSeedArg = true
+			break
+		}
+	}
+	// Random seed has to be propagated to child processes.
+	if !foundRandomSeedArg {
+		commonArgs = append(commonArgs, fmt.Sprintf("--%s=%v", randomSeedFlagKey, suiteConfig.RandomSeed))
+	}
+
```

Comment:
nit: a little bit shorter
```suggestion
// Propagate seed to the childs.
	if !helpers.Contains(commonArgs, func(arg string) bool {
		return strings.HasPrefix(arg, fmt.Sprintf("--%s", randomSeedFlagKey))
	}) {
		commonArgs = append(commonArgs, fmt.Sprintf("--%s=%v", randomSeedFlagKey, suiteConfig.RandomSeed))
	}
```